--- arpwatch-2.1a15/Makefile.in.orig	2000-06-15 00:39:55.000000000 +0000
+++ arpwatch-2.1a15/Makefile.in	2019-09-27 08:52:18.000000000 +0000
@@ -31,7 +31,7 @@
 # Pathname of directory to install the man page
 MANDEST = @mandir@
 # Pathname of directory to install database file
-ARPDIR = $(prefix)/arpwatch
+ARPDIR = /var/lib/arpwatch
 
 # VPATH
 srcdir = @srcdir@
@@ -110,14 +110,12 @@
 	$(CC) $(CFLAGS) -o $@ zap.o intoa.o -lutil
 
 install: force
-	$(INSTALL) -m 555 -o bin -g bin arpwatch $(DESTDIR)$(BINDEST)
-	$(INSTALL) -m 555 -o bin -g bin arpsnmp $(DESTDIR)$(BINDEST)
+	$(INSTALL) -Dm 755 arpwatch $(DESTDIR)$(BINDEST)/arpwatch
+	$(INSTALL) -Dm 755 arpsnmp  $(DESTDIR)$(BINDEST)/arpsnmp
 
 install-man: force
-	$(INSTALL) -m 444 -o bin -g bin $(srcdir)/arpwatch.8 \
-	    $(DESTDIR)$(MANDEST)/man8
-	$(INSTALL) -m 444 -o bin -g bin $(srcdir)/arpsnmp.8 \
-	    $(DESTDIR)$(MANDEST)/man8
+	$(INSTALL) -Dm 644 $(srcdir)/arpwatch.8 $(DESTDIR)$(MANDEST)/man8/arpwatch.8
+	$(INSTALL) -Dm 644 $(srcdir)/arpsnmp.8  $(DESTDIR)$(MANDEST)/man8/arpsnmp.8
 
 lint:	$(GENSRC) force
 	lint -hbxn $(SRC) | \
