Use standard icons, drop gnome-flavored ones.

--- a/plugins/netstatus/netstatus-icon.c
+++ b/plugins/netstatus/netstatus-icon.c
@@ -288,55 +288,55 @@
 {
   netstatus_icon_init_pixbuf (icon,
 			      &icon->priv->icons [NETSTATUS_STATE_DISCONNECTED],
-			      "gnome-netstatus-disconn");
+			      "network-offline");
   icon->priv->scaled_icons [NETSTATUS_STATE_DISCONNECTED] = NULL;
 
   netstatus_icon_init_pixbuf (icon,
 			      &icon->priv->icons [NETSTATUS_STATE_IDLE],
-			      "gnome-netstatus-idle");
+			      "network-idle");
   icon->priv->scaled_icons [NETSTATUS_STATE_IDLE] = NULL;
 
   netstatus_icon_init_pixbuf (icon,
 			      &icon->priv->icons [NETSTATUS_STATE_TX],
-			      "gnome-netstatus-tx");
+			      "network-transmit");
   icon->priv->scaled_icons [NETSTATUS_STATE_TX] = NULL;
 
   netstatus_icon_init_pixbuf (icon,
 			      &icon->priv->icons [NETSTATUS_STATE_RX],
-			      "gnome-netstatus-rx");
+			      "network-receive");
   icon->priv->scaled_icons [NETSTATUS_STATE_RX] = NULL;
 
   netstatus_icon_init_pixbuf (icon,
 			      &icon->priv->icons [NETSTATUS_STATE_TX_RX],
-			      "gnome-netstatus-txrx");
+			      "network-transmit-receive");
   icon->priv->scaled_icons [NETSTATUS_STATE_TX_RX] = NULL;
 
   netstatus_icon_init_pixbuf (icon,
 			      &icon->priv->icons [NETSTATUS_STATE_ERROR],
-			      "gnome-netstatus-error");
+			      "network-error");
   icon->priv->scaled_icons [NETSTATUS_STATE_ERROR] = NULL;
 
   netstatus_icon_init_pixbuf (icon,
 			      &icon->priv->signal_icons [NETSTATUS_SIGNAL_0_24],
-			      "gnome-netstatus-0-24");
+			      "network-wireless-signal-none");
   icon->priv->rotated_signal_icons [NETSTATUS_SIGNAL_0_24] = NULL;
   icon->priv->scaled_signal_icons  [NETSTATUS_SIGNAL_0_24] = NULL;
 
   netstatus_icon_init_pixbuf (icon,
 			      &icon->priv->signal_icons [NETSTATUS_SIGNAL_25_49],
-			      "gnome-netstatus-25-49");
+			      "network-wireless-signal-low");
   icon->priv->rotated_signal_icons [NETSTATUS_SIGNAL_25_49] = NULL;
   icon->priv->scaled_signal_icons  [NETSTATUS_SIGNAL_25_49] = NULL;
 
   netstatus_icon_init_pixbuf (icon,
 			      &icon->priv->signal_icons [NETSTATUS_SIGNAL_50_74],
-			      "gnome-netstatus-50-74");
+			      "network-wireless-signal-good");
   icon->priv->rotated_signal_icons [NETSTATUS_SIGNAL_50_74] = NULL;
   icon->priv->scaled_signal_icons  [NETSTATUS_SIGNAL_50_74] = NULL;
 
   netstatus_icon_init_pixbuf (icon,
 			      &icon->priv->signal_icons [NETSTATUS_SIGNAL_75_100],
-			      "gnome-netstatus-75-100");
+			      "network-wireless-signal-excellent");
   icon->priv->rotated_signal_icons [NETSTATUS_SIGNAL_75_100] = NULL;
   icon->priv->scaled_signal_icons  [NETSTATUS_SIGNAL_75_100] = NULL;
 
