--- gpicview-0.2.5/po/Makefile.in.in.orig	2016-02-20 22:40:27.000000000 +0000
+++ gpicview-0.2.5/po/Makefile.in.in	2019-10-04 10:50:01.367888727 +0000
@@ -34,7 +34,7 @@
 datarootdir = @datarootdir@
 libdir = @libdir@
 DATADIRNAME = @DATADIRNAME@
-itlocaledir = $(prefix)/$(DATADIRNAME)/locale
+itlocaledir = $(datadir)/locale
 subdir = po
 install_sh = @install_sh@
 # Automake >= 1.8 provides @mkdir_p@.
