--- leafpad-0.8.18.1/po/Makefile.in.in.orig	2010-12-23 13:56:38.000000000 +0000
+++ leafpad-0.8.18.1/po/Makefile.in.in	2019-10-04 11:05:24.738797941 +0000
@@ -34,7 +34,7 @@
 datarootdir = @datarootdir@
 libdir = @libdir@
 DATADIRNAME = @DATADIRNAME@
-itlocaledir = $(prefix)/$(DATADIRNAME)/locale
+itlocaledir = $(datadir)/locale
 subdir = po
 install_sh = @install_sh@
 # Automake >= 1.8 provides @mkdir_p@.
