--- libwnck-2.31.0/po/Makefile.in.in.orig	2012-03-05 09:30:49.000000000 +0000
+++ libwnck-2.31.0/po/Makefile.in.in	2019-10-03 10:47:45.054787511 +0000
@@ -34,7 +34,7 @@
 datarootdir = @datarootdir@
 libdir = @libdir@
 DATADIRNAME = @DATADIRNAME@
-itlocaledir = $(prefix)/$(DATADIRNAME)/locale
+itlocaledir = $(datadir)/locale
 subdir = po
 install_sh = @install_sh@
 # Automake >= 1.8 provides @mkdir_p@.
