--- gtk-engines-2.20.2/themes/Clearlooks/gtk-2.0/gtkrc.orig	2010-09-23 13:40:24.000000000 +0000
+++ gtk-engines-2.20.2/themes/Clearlooks/gtk-2.0/gtkrc	2019-10-02 12:27:33.701320046 +0000
@@ -1,7 +1,7 @@
 
 # Please keep this gtkrc in sync with the other ones from Clearlooks based themes.
 
-gtk-color-scheme = "base_color:#ffffff\nfg_color:#000000\ntooltip_fg_color:#000000\nselected_bg_color:#86ABD9\nselected_fg_color:#ffffff\ntext_color:#1A1A1A\nbg_color:#EDECEB\ntooltip_bg_color:#F5F5B5"
+gtk-color-scheme = "base_color:#fff\nfg_color:#000\ntooltip_fg_color:#000\nselected_bg_color:#86ABD9\nselected_fg_color:#fff\ntext_color:#1A1A1A\nbg_color:#EDECEB\ntooltip_bg_color:#F5F5B5"
 
 style "default" {
 	xthickness = 1
--- gtk-engines-2.20.2/themes/Crux/gtk-2.0/gtkrc.orig	2010-09-23 13:40:24.000000000 +0000
+++ gtk-engines-2.20.2/themes/Crux/gtk-2.0/gtkrc	2019-10-02 12:24:47.451703023 +0000
@@ -32,8 +32,7 @@
 gtk_color_scheme = "fg_color:#000;bg_color:#d3d7cf;base_color:#fff;text_color:#000;selected_bg_color:#75507b;selected_fg_color:#fff;tooltip_bg_color:#F5F5B5;tooltip_fg_color:#000"
 
 
-style "default"
-{
+style "default" {
   fg[NORMAL]        = @fg_color           # was 000000 --> OK
   fg[PRELIGHT]      = @fg_color           # was 000000 --> OK
   fg[SELECTED]      = @selected_fg_color  # was ffffff --> OK
@@ -83,8 +82,7 @@
 class "GtkWidget" style "default"
 
 # Style used for (prelighted) menu items
-style "menuitem"
-{
+style "menuitem" {
   fg[PRELIGHT] = @selected_fg_color # was 000000 --> OK
   bg[PRELIGHT] = @selected_bg_color # was 75507b --> OK
 }
@@ -93,24 +91,21 @@
 widget_class "*.*MenuItem.*" style "menuitem"
 
 
-style "menu"
-{
+style "menu" {
   xthickness = 3
 }
 
 # This enables gradiented menu backgrounds
 class "GtkMenu" style "menu"
 
-style "entry"
-{
+style "entry" {
   xthickness = 3
   ythickness = 3
 
   # Slightly lighter so that the selection will be visible properly
   bg[SELECTED] = mix(0.85, @selected_bg_color, @base_color)
 
-  engine "crux-engine"
-  {
+  engine "crux-engine" {
     thickness = 3
   }
 }
@@ -119,23 +114,20 @@
 class "GtkEntry" style "entry"
 class "GtkText" style "entry"
 
-style "trough"
-{
+style "trough" {
    bg[NORMAL] = shade (0.9, @bg_color)
    # fg[PRELIGHT] is used for the text underneath the bar.
    fg[PRELIGHT] = @selected_fg_color
 }
 class "GtkProgressBar" style "trough"
 
-style "statusbar"
-{
+style "statusbar" {
    xthickness = 3
    ythickness = 3
 }
 class "GtkStatusbar" style "statusbar"
 
-style "crux-tooltips"
-{
+style "crux-tooltips" {
         xthickness = 4
         ythickness = 4
 
@@ -158,8 +150,7 @@
 # appears-as-list mode.
 # This style does not affect GtkComboBoxEntry, it does have an effect
 # on comboboxes in appears-as-list mode though.
-style "crux-text-is-fg-color-workaround"
-{
+style "crux-text-is-fg-color-workaround" {
 	text[NORMAL]        = @fg_color
 	text[PRELIGHT]      = @fg_color
 	text[SELECTED]      = @selected_fg_color
@@ -168,8 +159,7 @@
 }
 widget_class "*.<GtkComboBox>.<GtkCellView>"   style "crux-text-is-fg-color-workaround"
 
-style "crux-menuitem-text-is-fg-color-workaround"
-{
+style "crux-menuitem-text-is-fg-color-workaround" {
 	text[NORMAL]        = @fg_color
 	text[PRELIGHT]      = @selected_fg_color
 	text[SELECTED]      = @selected_fg_color
@@ -181,8 +171,7 @@
 # Work around the usage of GtkLabel inside GtkListItems to display text.
 # This breaks because the label is shown on a background that is based on the
 # base color set.
-style "crux-fg-is-text-color-workaround"
-{
+style "crux-fg-is-text-color-workaround" {
 	fg[NORMAL]      = @text_color
 	fg[PRELIGHT]    = @text_color
 	fg[ACTIVE]      = @selected_fg_color
--- gtk-engines-2.20.2/themes/Industrial/gtk-2.0/gtkrc.orig	2010-09-23 13:40:24.000000000 +0000
+++ gtk-engines-2.20.2/themes/Industrial/gtk-2.0/gtkrc	2019-10-02 13:15:11.402669399 +0000
@@ -1,7 +1,6 @@
-gtk-color-scheme = "bg_color: #f6f6f6\nfg_color: #000\nbase_color: #fff\ntext_color: #000\nselected_bg_color: #6484a4\nselected_fg_color: #fff\ntooltip_bg_color: #e5e5e5\ntooltip_fg_color: #000000"
+gtk-color-scheme = "bg_color:#f6f6f6\nfg_color:#000\nbase_color:#fff\ntext_color:#000\nselected_bg_color:#6484a4\nselected_fg_color:#fff\ntooltip_bg_color:#e5e5e5\ntooltip_fg_color:#000"
 
-style "industrial-default"
-{
+style "industrial-default" {
   xthickness = 1
   ythickness = 1
 
@@ -52,16 +51,16 @@
   fg[PRELIGHT]		= @fg_color
   fg[ACTIVE]		= @fg_color
   fg[SELECTED]		= @selected_fg_color
-  fg[INSENSITIVE]	= mix (0.4, @fg_color, shade (0.85, @bg_color)) #shaded to bg[INSENSITIVE]
+  fg[INSENSITIVE]	= mix (0.4, @fg_color, shade (0.85, @bg_color)) # shaded to bg[INSENSITIVE]
 
   base[NORMAL]		= @base_color
-  base[PRELIGHT]	= @base_color #XXX
-  base[ACTIVE]		= mix (0.45, @selected_bg_color, @base_color) #b6c9cf
+  base[PRELIGHT]	= @base_color # XXX
+  base[ACTIVE]		= mix (0.45, @selected_bg_color, @base_color) # b6c9cf
   base[SELECTED]	= @selected_bg_color
   base[INSENSITIVE]	= mix (0.95, @base_color, @text_color)
 
   text[NORMAL]		= @text_color
-  text[PRELIGHT]	= @text_color #XXX
+  text[PRELIGHT]	= @text_color # XXX
   text[ACTIVE]		= @selected_fg_color
   text[SELECTED]	= @selected_fg_color
   text[INSENSITIVE]	= mix (0.5, @text_color, @base_color)
@@ -83,29 +82,25 @@
 #  #base[NORMAL]  = "#b6c9cf"
 #  #text[NORMAL]  = "#000000"
 #
-  engine "industrial"
-  {
+  engine "industrial" {
     # Set a hint to disable backward compatibility fallbacks.
     hint = "use-hints"
   }
 }
 
 # Style to apply an exterior forcus to GtkEntry
-style "industrial-exterior-focus"
-{
+style "industrial-exterior-focus" {
 	GtkWidget::interior-focus = 0
 	GtkWidget::focus-line-width = 0
 }
 
 # Add trough side details to GtkScale
-style "industrial-scale"
-{
+style "industrial-scale" {
 	GtkRange::trough-side-details = 1
 }
 
 # Style for the entry progress bar -- ie. change the colors
-style "industrial-entry"
-{
+style "industrial-entry" {
   xthickness = 2
   ythickness = 2
 
@@ -114,36 +109,30 @@
 }
 
 # generic squared buttons style, used for the tasklist (treeview is in the extra treeview style)
-style "industrial-squared"
-{
-  engine "industrial" 
-  {
+style "industrial-squared" {
+  engine "industrial" {
     rounded_buttons = FALSE
   }
 }
 
-style "industrial-wide"
-{
+style "industrial-wide" {
   xthickness = 2
   ythickness = 2
 }
 
 # style to remove the border around the progress bar
-style "industrial-progress"
-{
+style "industrial-progress" {
   xthickness = 0
   ythickness = 0
 }
 
 # make scrollbar arrows lighter
-style "industrial-arrows"
-{
+style "industrial-arrows" {
   fg[NORMAL] = mix (0.8, @fg_color, @bg_color)
 }
 
 # menuitems
-style "industrial-menuitem"
-{
+style "industrial-menuitem" {
   xthickness = 3
   ythickness = 3
 
@@ -153,37 +142,31 @@
   fg[SELECTED]  = @selected_fg_color
 }
 
-style "industrial-menubar"
-{
+style "industrial-menubar" {
   xthickness = 3
   ythickness = 3
 }
 
-style "industrial-tree-header"
-{
-  engine "industrial" 
-  {
+style "industrial-tree-header" {
+  engine "industrial" {
     hint = "treeview-header"
   }
   xthickness = 2
   ythickness = 2
 }
 
-style "industrial-tooltips"
-{
+style "industrial-tooltips" {
   xthickness = 4
   ythickness = 4
   bg[NORMAL] = @tooltip_bg_color
   fg[NORMAL] = @tooltip_fg_color
 }
 
-style "industrial-druid"
-{
+style "industrial-druid" {
   bg[SELECTED] = @selected_bg_color
 }
 
-style "metacity-frame"
-{
+style "metacity-frame" {
 
 	# Normal base color
  	bg[NORMAL]      = @bg_color
@@ -204,18 +187,14 @@
 	fg[SELECTED]	= @selected_fg_color
 }
 
-style "industrial-spinbutton"
-{
-	engine "industrial"
-	{
+style "industrial-spinbutton" {
+	engine "industrial" {
 		hint	= "spinbutton"
 	}
 }
 
-style "industrial-comboboxentry"
-{
-	engine "industrial"
-	{
+style "industrial-comboboxentry" {
+	engine "industrial" {
 		hint	= "comboboxentry"
 	}
 }
@@ -269,8 +248,7 @@
 # appears-as-list mode.
 # This style does not affect GtkComboBoxEntry, it does have an effect
 # on comboboxes in appears-as-list mode though.
-style "industrial-text-is-fg-color-workaround"
-{
+style "industrial-text-is-fg-color-workaround" {
 	text[NORMAL]        = @fg_color
 	text[PRELIGHT]      = @fg_color
 	text[SELECTED]      = @selected_fg_color
@@ -279,8 +257,7 @@
 }
 widget_class "*.<GtkComboBox>.<GtkCellView>"   style "industrial-text-is-fg-color-workaround"
 
-style "industrial-menuitem-text-is-fg-color-workaround"
-{
+style "industrial-menuitem-text-is-fg-color-workaround" {
 	text[NORMAL]        = @fg_color
 	text[PRELIGHT]      = @selected_fg_color
 	text[SELECTED]      = @selected_fg_color
@@ -292,8 +269,7 @@
 # Work around the usage of GtkLabel inside GtkListItems to display text.
 # This breaks because the label is shown on a background that is based on the
 # base color set.
-style "industrial-fg-is-text-color-workaround"
-{
+style "industrial-fg-is-text-color-workaround" {
 	fg[NORMAL]      = @text_color
 	fg[PRELIGHT]    = @text_color
 	fg[ACTIVE]      = @selected_fg_color
--- gtk-engines-2.20.2/themes/Mist/gtk-2.0/gtkrc.orig	2010-09-23 13:40:24.000000000 +0000
+++ gtk-engines-2.20.2/themes/Mist/gtk-2.0/gtkrc	2019-10-02 12:38:13.924679551 +0000
@@ -1,11 +1,9 @@
-gtk-color-scheme =
-"bg_color:#eaeaea\nfg_color:#000\nbase_color:#fff\ntext_color:#000\nselected_fg_color:#fff\nselected_bg_color:#729fcf"
+gtk-color-scheme = "bg_color:#eaeaea\nfg_color:#000\nbase_color:#fff\ntext_color:#000\nselected_fg_color:#fff\nselected_bg_color:#729fcf"
 
-style "default"
-{
+style "default" {
         fg[NORMAL]              = @fg_color
         fg[ACTIVE]              = @fg_color
-        fg[INSENSITIVE]         = mix (0.4, @fg_color, shade (0.85, @bg_color)) #shaded to bg[INSENSITIVE]
+        fg[INSENSITIVE]         = mix (0.4, @fg_color, shade (0.85, @bg_color)) # shaded to bg[INSENSITIVE]
 
         fg[PRELIGHT]            = @fg_color
         fg[SELECTED]            = @selected_fg_color
@@ -52,27 +50,23 @@
 	GtkMenu::horizontal_padding=0
 	GtkMenu::vertical_padding=0
 
-	engine "mist"
- 	{
+	engine "mist" {
 	}
 }
 
-style "menuitem"
-{
+style "menuitem" {
 	ythickness = 2
 	xthickness = 2
 
         fg[PRELIGHT]          = @selected_fg_color
 }
 
-style "menu" 
-{
+style "menu" {
 	ythickness = 2
 	xthickness = 2
 } 
 
-style "entry"
-{
+style "entry" {
 	bg[SELECTED] = mix(0.35, @selected_bg_color, @base_color)
 	fg[SELECTED] = @text_color
 }
@@ -90,21 +84,19 @@
 # appears-as-list mode.
 # This style does not affect GtkComboBoxEntry, it does have an effect
 # on comboboxes in appears-as-list mode though.
-style "mist-text-is-fg-color-workaround"
-{
+style "mist-text-is-fg-color-workaround" {
         text[NORMAL]              = @fg_color
         text[ACTIVE]              = @fg_color
-        text[INSENSITIVE]         = mix (0.4, @fg_color, shade (0.85, @bg_color)) #shaded to bg[INSENSITIVE]
+        text[INSENSITIVE]         = mix (0.4, @fg_color, shade (0.85, @bg_color)) # shaded to bg[INSENSITIVE]
         text[PRELIGHT]            = @fg_color
         text[SELECTED]            = @selected_fg_color
 }
 widget_class "*.<GtkComboBox>.<GtkCellView>"   style "mist-text-is-fg-color-workaround"
 
-style "mist-menuitem-text-is-fg-color-workaround"
-{
+style "mist-menuitem-text-is-fg-color-workaround" {
         text[NORMAL]              = @fg_color
         text[ACTIVE]              = @fg_color
-        text[INSENSITIVE]         = mix (0.4, @fg_color, shade (0.85, @bg_color)) #shaded to bg[INSENSITIVE]
+        text[INSENSITIVE]         = mix (0.4, @fg_color, shade (0.85, @bg_color)) # shaded to bg[INSENSITIVE]
         text[PRELIGHT]            = @selected_fg_color
         text[SELECTED]            = @selected_fg_color
 }
@@ -113,8 +105,7 @@
 # Work around the usage of GtkLabel inside GtkListItems to display text.
 # This breaks because the label is shown on a background that is based on the
 # base color set.
-style "mist-fg-is-text-color-workaround"
-{
+style "mist-fg-is-text-color-workaround" {
         fg[NORMAL]            = @text_color
         fg[ACTIVE]            = @text_color
         fg[PRELIGHT]          = @text_color
--- gtk-engines-2.20.2/themes/Redmond/gtk-2.0/gtkrc.orig	2010-09-23 13:40:24.000000000 +0000
+++ gtk-engines-2.20.2/themes/Redmond/gtk-2.0/gtkrc	2019-10-02 12:42:32.930425389 +0000
@@ -1,8 +1,7 @@
 
 gtk-icon-sizes = "mini-commander-icon=32,32:print-manager=32,32:panel-button=32,32:gtk-dnd=32,32:gtk-menu=16,16:panel-menu=22,22:gtk-large-toolbar=16,16:gtk-small-toolbar=16,16:gtk-button=16,16:gtk-dialog=32,32"
 
-style "standard-default"
-{
+style "standard-default" {
   GtkWidget::interior_focus = 2
   GtkButton::default_border = { 1, 1, 1, 1 }
   GtkButton::default_outside_border = { 0, 0, 0, 0 }
@@ -50,33 +49,29 @@
 
 class "GtkWidget" style "standard-default"
 
-style "standard-menu"
-{
+style "standard-menu" {
   bg[PRELIGHT] = { 0.04, 0.14, 0.41 }
   fg[PRELIGHT] = { 1.0, 1.0, 1.0 }
 }
 
 widget_class "*MenuItem*" style "standard-menu"
 
-style "standard-menubar-item"
-{
-  bg[PRELIGHT] = { 0.83, 0.81, 0.78 } 
+style "standard-menubar-item" {
+  bg[PRELIGHT] = { 0.83, 0.81, 0.78 }
   fg[PRELIGHT] = { 0.0, 0.0, 0.0 }
 }
 
 class "*MenuBar.*MenuItem*" style "standard-menubar-item"
 widget_class "*MenuBar.*MenuItem*" style "standard-menubar-item"
 
-style "standard-list"
-{
+style "standard-list" {
    bg[NORMAL] = { 0.9, 0.9, 0.9 }
    bg[PRELIGHT] = { 0.0, 0.0, 0.0 }
 }
 
 class "GtkList" style "standard-list"
 
-style "standard-scrollbar"
-{
+style "standard-scrollbar" {
   GtkRange::trough_border = 0
   GtkRange::slider_width = 16
   GtkRange::stepper_size = 16
@@ -86,16 +81,14 @@
 class "GtkScrollbar" style "standard-scrollbar"
 
 
-style "standard-scales"
-{
+style "standard-scales" {
   GtkRange::slider-width = 21
   GtkScale::slider-length = 11
 }
 
 class "*Scale*" style "standard-scales"
 
-style "standard-option-menu"
-{
+style "standard-option-menu" {
   GtkOptionMenu::indicator_width = 7
   GtkOptionMenu::indicator_left_spacing = 6
   GtkOptionMenu::indicator_right_spacing = 4
@@ -109,8 +102,7 @@
 class "GtkProgressBar" style "standard-progressbar"
 
 
-style "standard-entry"
-{
+style "standard-entry" {
   bg[SELECTED] = "#1C40A8"
   fg[SELECTED] = { 1.0, 1.0, 1.0 }
 }
--- gtk-engines-2.20.2/themes/ThinIce/gtk-2.0/gtkrc.orig	2010-09-23 13:40:24.000000000 +0000
+++ gtk-engines-2.20.2/themes/ThinIce/gtk-2.0/gtkrc	2019-10-02 12:46:17.161764396 +0000
@@ -1,6 +1,5 @@
 # Edit these colors and fonts however you like.
-style "default"
-{
+style "default" {
   GtkOptionMenu::indicator_spacing = { 3, 2, 1, 1 }
   GtkEntry::progress-border = { 1, 1, 1, 1 }
   GtkRange::slider_width = 11
