--- lxrandr-0.3.1/po/Makefile.in.in.orig	2016-02-21 00:03:53.000000000 +0000
+++ lxrandr-0.3.1/po/Makefile.in.in	2019-10-04 09:31:13.924651059 +0000
@@ -34,7 +34,7 @@
 datarootdir = @datarootdir@
 libdir = @libdir@
 DATADIRNAME = @DATADIRNAME@
-itlocaledir = $(prefix)/$(DATADIRNAME)/locale
+itlocaledir = $(datadir)/locale
 subdir = po
 install_sh = @install_sh@
 # Automake >= 1.8 provides @mkdir_p@.
